portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,34 @@
+body {
+    background-color: #cccac9;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+#footer {
```

Comment:
Note that footers in CSS can be a bit weird.

With this setup, the footer will always be at the bottom of the screen, not the bottom of the page, and will therefore make the visible section of the content smaller. Additionally, even if you do want a footer that's always on screen, you'd need to add an element to "clear" space for the footer at the bottom of the main content. That's because the scrollbar won't account for the footer, so if the content is long enough to reach the space the footer occupies, the footer will actually cover the last couple lines of the content.

The way I do it on my personal website is like this: https://www.freecodecamp.org/news/how-to-keep-your-footer-where-it-belongs-59c6aa05c59c/#:~:text=The%20footer%20is%20set%20to,the%20content%2Dwrap%20above%20it.

Reply:
I see the difference, I implemented the adjustments. I don't think my homepage will have enough content for scroll issues or content being blocked but I added just in case

Reply:
You never know how big a user's browser window or screen will be. Something may well fit nicely when you run it on your screen, but could become horribly mangled on smaller screens, especially mobile browsers.